worlds/stardew_valley/test/TestOptions.py

```diff
@@ -184,7 +188,8 @@ def test_given_island_related_goal_then_override_exclude_ginger_island(self):
                     multiworld = setup_solo_multiworld(
                         {options.Goal.internal_name: options.Goal.options[goal],
                             island_option.internal_name: island_option.options[value]})
-                    self.assertEqual(multiworld.worlds[self.player].options[island_option.internal_name], island_option.option_false)
+                    stardew_world: StardewValleyWorld = multiworld.worlds[self.player]
+                    self.assertEqual(stardew_world.options.exclude_ginger_island, island_option.option_false)
```

Comment:
This makes me think we should probably make getting the world and option results easily within unit tests would be beneficial. I'll probably do that as a second step since it's a bit out of scope.